src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -134,9 +134,9 @@ public function getBasketsAction(ParamFetcherInterface $paramFetcher)
      *  }
      * )
      *
-     * @View(serializerGroups={"sonata_api_read"}, serializerEnableMaxDepthChecks=true)
+     * @Rest\View(serializerGroups={"sonata_api_read"}, serializerEnableMaxDepthChecks=true)
      *
-     * @param $id
+     * @param int $id Basket identifier
```

Comment:
Is the model forcing the basket to allow only integer ids?

Reply:
Model allow for `mixed`. I use `int` to be unequivocal with `@ApiDoc`

Reply:
There is an issue with the `@ApiDoc` annotation then. I think we should not propagate the same problem here.
I understand you're trying to be consistent, but in this case the reference type does not seem to be the right choice.

Reply:
I reverted it (docblock type hints). Lets do it in seperate PR.